junction/profiles/views.py

```diff
@@ -22,3 +29,26 @@ def dashboard(request):
                 conf_proposals[conf.name] = [proposal]
     return render(request, 'profiles/dashboard.html',
                   {'conf_proposals': conf_proposals})
+
+
+@login_required
+def profile(request):
+    username = request.user
+    detail = None
+
+    if request.method == "POST" and username == request.user:
+        user = get_object_or_404(User, pk=username.id)
+        detail = get_object_or_404(Profile, user=user)
+        detail_form = ProfileForm(request.POST, instance=detail)
+
+        if detail_form.is_valid():
+            detail = detail_form.save()
+            return HttpResponseRedirect(reverse('profiles:dashboard'))
+
+    elif request.method == "GET":
+        user = get_object_or_404(User, pk=username.id)
+        detail = get_object_or_404(Profile, user=user)
```

Comment:
@ChillarAnand It is first checking that the object exists or not and then if it is found it returns in the context else creates a new profile.
Please explain if I am wrong.